src/services/axios.ts

```
@@ -62,10 +62,11 @@ axiosInstance.interceptors.response.use(
 
       // If a refresh is already in progress, queue the original request
       if (isRefreshing) {
-        return new Promise(function(resolve, reject) {
+        return new Promise(function (resolve, reject) {
           failedQueue.push({ resolve, reject });
         }).then(token => {
           // When refresh is done, retry the original request with the new token
+          debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)